class DomainError(Exception):
    pass


class ResourceNotFound(DomainError):
    pass
